keras/api/_tf_keras/keras/ops/__init__.py

```diff
@@ -219,6 +219,7 @@
 from keras.src.ops.numpy import split
 from keras.src.ops.numpy import sqrt
 from keras.src.ops.numpy import square
+from keras.src.ops.numpy import squared_difference
```

Comment:
Does this op have an equivalent in the NumPy API? `np.squared_difference` doesn't seem to be a thing, so if we add such an op it would either have to follow the name used by NumPy, or not be in the numpy namespace.

Reply:
No, there is no NumPy equivalent.

But backend implementation for all platforms except TF will require to import `square` and `subtract` from `numpy` namespace inside backend functions to overcome possible circular dependency.